test/src/htmlfilesetserv/test/HTMLFileSetServServerTest.java

```diff
@@ -293,17 +293,6 @@ private static void loadTestData() throws Exception {
 		// KBaseReport.Report testing
 		final BasicShockClient bsc = new BasicShockClient(SHOCK_URL, TOKEN1);
 		
-		// empty nodes will not be allowed in the Blobstore but for now...
-		final ShockNode emptynode = bsc.addNode(toBAOS(""), 0, "fn", "fmt");
-		final String emptyhandle = makeHandle(emptynode);
-		CREATED_NODES.put(emptynode.getId().getId(),
-				new NodeAndHandle(emptynode, emptyhandle));
-		
-		final ShockNode delnode = bsc.addNode(toBAOS(""), 0, "fn", "fmt");
```

Comment:
The tests for the deleted node should still be performed - just make the file non-zero length and they should work I think